src/components/account-header/account-header.yaml

```diff
@@ -87,6 +87,7 @@ examples:
       classes: 'the-header-classes'
   visualRegressionTests:
     backstopScenarioOptions:
+      misMatchThreshold: 0.1
```

Comment:
for some reason, even though tests are being done in docker container, aand katie is on linux like jenkins, in this test, the account menu hover on language switcher has a 0.03 change in pixels (underline is lower in one than other) https://build.tax.service.gov.uk/job/PlatUI/job/Libraries/job/hmrc-frontend-pr-builder/193/artifact/backstop_data/html_report/index.html and search for:
```
account-header (link-states-on-desktop example) (language-select-current-lang-focused-and-alternate-lang-hovered)
```
guessing we might not need this anymore once we enable the new link styles, or this could be something we could fix by reviewing our custom link styles for this and figuring out what is causing this difference in rendering

Reply:
So are happy to get this change in then? looks good to me.

Reply:
yup, it's it's just the underline moving a pixel down - guess maybe some number rounding thing or something like that - no issues for anyone using the component